.ci/pytorch/build.sh

```diff
@@ -241,13 +241,16 @@ if [[ "$BUILD_ENVIRONMENT" == *-bazel-* ]]; then
     tools/bazel build --config=no-tty "${BAZEL_MEM_LIMIT}" "${BAZEL_CPU_LIMIT}" //...
   fi
 else
+  git config --global --add safe.directory /var/lib/jenkins/workspace
```

Comment:
Should this be part of docker build (I remember adding this one in the past, see https://github.com/pytorch/pytorch/issues/78659#issuecomment-1144107327 )